code/modules/shuttle/supply.dm

```diff
@@ -211,9 +217,8 @@
 		credits_to_deposit += credits_from_intel
 
 	if(crate_count > 0)
-		var/credits_from_crates = crate_count * SSeconomy.credits_per_crate
```

Comment:
credits_per_crate is used nowhere now with this change, if you want to remove it

Reply:
removed